cyral/resource_cyral_integration_aws_iam_authn.go

```diff
@@ -0,0 +1,179 @@
+package cyral
```

Comment:
@VictorGFM @Yowgf I still have to add unit tests, but I wanted some early feedback before doing that :grin: